docs/source/reference/changelog.md

```diff
@@ -13,6 +13,14 @@ command line for details.
 ### 4.0.0 - 2023-04-20
 
 4.0 is a major release, but a small one.
+
+:::{note}
+**Upgrade note**
```

Comment:
```suggestion
:::{admonition} Upgrade note
```

Like this, we get a note section with the title "Upgrade note" as compared to having it titled "Note" followed by the bold text "Upgrade note" inside it.

Reply:
![image](https://user-images.githubusercontent.com/3837114/233358489-ec487aab-34a2-4cd6-9fe8-0d9b86f542de.png)

Compared to

![image](https://user-images.githubusercontent.com/3837114/233358952-2e1035ec-d26a-4673-8865-8a8c18c832db.png)

Reply:
Either seems fine, merge directly if you want